platform/darwin/src/MGLStyle.mm

```diff
@@ -609,4 +610,59 @@ - (NSString *)description
             self.URL ? [NSString stringWithFormat:@"\"%@\"", self.URL] : self.URL];
 }
 
+#pragma mark Style language preferences
+
+- (void)setLocalizeLabels:(BOOL)localizeLabels
+{
+    _localizeLabels = localizeLabels;
+    
+    if (_localizeLabels) {
```

Comment:
Normally, a custom setter checks whether the old value is equal to the new value. If it is, the setter returns early because there’s nothing to do.